crates/noirc_evaluator/src/ssa_refactor/acir_gen/acir_ir/generated_acir.rs

```diff
@@ -357,6 +348,29 @@ impl GeneratedAcir {
         Ok((q_witness, r_witness))
     }
 
+    pub(crate) fn brillig_quotient(
+        &mut self,
+        lhs: &Expression,
+        rhs: &Expression,
+        predicate: &Expression,
+        max_bit_size: u32,
```

Comment:
In the brillig_vm, the maximum bit_size for binary operations that we support is currently 127 bits, is it guaranteed that the quotient max_bit_size will never exceed 127 bits?

Reply:
No, it is not guaranteed, you should support any bit size.